releasenotes/v1.8.6.md

```diff
@@ -0,0 +1,18 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.8.5
+
+## :bug: Bug Fixes
+
+- bump golang to 1.23.6 (#2323)
+
+## :seedling: Others
+
+- Bump IPAM to 1.8.5 (#2345)
+- Bump the capi group across 3 directories with 2 updates (#2341)
+- Bump the kubernetes group across 4 directories with 6 updates (#2340)
```

Comment:
Can you update this to match the current PR title? For some reason, our automation takes the original title here, and not the current one.